src/Microsoft.Extensions.FileProviders.Physical/PhysicalFileInfo.cs

```diff
@@ -29,13 +29,12 @@ public PhysicalFileInfo(FileInfo info)
 
         public Stream CreateReadStream()
         {
-            // Note: Buffer size must be greater than zero, even if the file size is zero.
             return new FileStream(
```

Comment:
Should we use the buffer pool here instead? Then we can use whatever sized buffer we like.